src/ImageSharp/Processing/Processors/Quantization/EuclideanPixelMap{TPixel}.cs

```diff
@@ -184,14 +184,14 @@ private static float DistanceSquared(Rgba32 a, Rgba32 b)
     /// The granularity of the cache has been determined based upon the current
     /// suite of test images and provides the lowest possible memory usage while
     /// providing enough match accuracy.
-    /// Entry count is currently limited to 4601025 entries (8MB).
+    /// Entry count is currently limited to 2335905 entries (4MB).
     /// </para>
     /// </remarks>
     private unsafe struct ColorDistanceCache : IDisposable
     {
         private const int IndexRBits = 5;
         private const int IndexGBits = 5;
-        private const int IndexBBits = 6;
+        private const int IndexBBits = 5;
```

Comment:
Is there a test that unveils that "bug" (was this a bug?)?

Reply:
I was something I just caught doing a visual test which is hard to test for. 

The cache was only filled to a maximum value of ~96% which led to matching where it shouldn't. This led to transparent areas in the frame following deduplication. Originally, I was going to simply add more precision for a single component (blue) which fixed the issue, but I discovered that with the new index determinaiton code the cache fills to a maximum of ~98% and the problem simply went away.